internal/users/manager_test.go

```diff
@@ -378,6 +465,8 @@ func TestUserByIDAndName(t *testing.T) {
 			// Registering a temporary user creates it with a random UID, GID, and gecos, so we have to make it
 			// deterministic before comparing it with the golden file
 			if tc.isTempUser {
+				require.True(t, strings.HasPrefix(user.Name, tempentries.UserPrefix))
+				user.Name = tempentries.UserPrefix + "-XXXXXX"
```

Comment:
```suggestion
				user.Name = tempentries.UserPrefix + "-{{name}}"
```

Reply:
given this does not get replaced, maybe instead of `name` we can use soemthing like `{{random-suffix}}`?